packages/vuepress-theme-odyssey/styles/components/_DocsSidebar.scss

```diff
@@ -0,0 +1,106 @@
+a.docs-site-title,
+a.docs-site-title:visited {
+  display: block;
+  padding: $spacing-xs $spacing-xs;
+  color: var(--docs-text-heading, #{$text-heading});
+  font-size: $size-title-4;
+  font-weight: 600;
+  text-decoration: none;
+  border: none;
+}
+
+@include mq(l) {
+  .is-sidebar-expanded {
+    html, body {
+      overflow-x: hidden;
+    }
+
+    body::before {
+      content: '';
+      position: fixed;
+      z-index: 10;
+      width: 100vw;
+      height: 100vh;
+      background: rgba(cv('gray', '900'), 0.75);
+    }
+  }
+}
+
+.docs-sidebar {
+  @include mq(l) {
+    visibility: hidden;
+    position: fixed;
+    z-index: 10;
+    transform: translateX(-100%);
+    transition: all 0.33s ease-in-out;
+
+    .is-sidebar-expanded & {
+      visibility: visible;
+      transform: translateX(0%);
+      transition: all 0.33s ease-in-out;
+    }
+  }
+
+  .docs-link {
+    border: none;
+  }
+
+  .docs-sidebar--action {
+    display: none;
+    position: absolute;
+    top: $spacing-s;
+    right: $spacing-s;
+
+    @include mq(l) {
+      display: block;
+    }
+  }
+
+  .docs-site-title {
+    @include mq(l) {
+      display: none;
+    }
+  }
+
+  .docs-sidebar--content {
+    display: grid;
+    position: sticky;
+    z-index: 4;
+    top: 0;
+    grid-row-gap: $spacing-s;
+    grid-template-areas: 'header' 'main' 'footer';
+    grid-template-rows: auto 1fr auto;
+    height: 100vh;
+    max-height: 100vh;
+    padding: $spacing-l;
+    background: var(--docs-sidebar-bg, #{$white});
+  }
+
+  .docs-sidebar--header {
+    grid-area: header;
+    padding: 0 $spacing-xs 0;
+  }
+
+  .docs-sidebar--main {
+    grid-area: main;
+    width: #{20.625rem - $spacing-l};
```

Comment:
This seems a little random?

Reply:
Could we use `min-content, max-content` on the grid declaration instead of having a width here?